internal/integration/helpers.go

```diff
@@ -186,6 +187,18 @@ func testServer(tb testing.TB) (*serverenv.ServerEnv, *http.Client) {
 	}
 	mux.Handle("/cleanup-exposure", cleanupExposureHandler)
 
+	// Key Rotation
+	keyRotationConfig := &keyrotation.Config{
```

Comment:
To test rotation, we we want to tune this down further? 
I'm guessing most of the time this test won't do anything.

Reply:
Yes! Sorry I backed that out - I am going to make a separate PR for the test.